### DISCUSSION TITLE
[FEATURE REQUEST] Thalamus items should be under group managment, same as faction stuff instead of despawning

### DISCUSSION THREAD

**Discussion by PanmanS** [FakeFishGames/Barotrauma#12925] [Ideas and suggestions]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

So basically while it is indeed not a bug, it is quite annoying when you cannot re-plan wreck if it consumed by thalamus, like you can with faction/outpost (although in-game it only used decoratively, but still!). I think it will greatly improve variety in wreck building scene!

### Reproduction steps

none lol

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.1.18.1 (Treacherous Tides)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
> it is quite annoying when you cannot re-plan wreck if it consumed by thalamus

I'm not sure what you mean by this. To me it seems like it works essentially identically to the group system used in the faction outpost, the only difference is that you toggle the entities on/off from a different menu.

**Comment by PanmanS** [comment]
Well it does, but in group system ya can show/change/hide decorations based on what faction running and I didnt try it yet, but it seems it is possible to expand it in way, that it will allow changing entire submarine structure (excluding hulls and gaps)


So basically if Im gonna have thalamus group and Ibwill some objects to it, I could make decorations more fitting for it like:
- removing conflicting decorations
- adding more "infected" looking look for props via using decorations by adding them in group of thalamus
- (if structures gonna be possible to add to group) alter the outside/inside layout with stuff more than just thalamus walls (maybe it is debris formed by thalamus spike growing upwards and so on)
- basically more control over how infected wreck looks like

**Comment by PanmanS** [comment]
But even standard group functions (which is hiding/making invisible deco of opposite faction in our case) would be still very useful for reasons stated above

**Comment by ubertpendragon** [comment]
I will simplify for Panman 

Thalamus (and abandoned outposts) use a system that is basically how groups work for coalition/separatists without being groups

![image](https://github.com/Regalis11/Barotrauma/assets/73229309/4c53307b-e4c0-41eb-903b-5052d44415c6)



Groups are good for making variants 

The idea would be to allow wrecks (and abandoned outposts) to select a group at random, with thalamus being one of those groups and affected by the thalamus% chance defined in biome

**Comment by PanmanS** [comment]
Ughhj, no

**Comment by PanmanS** [comment]
My whole thing is in making some items and deco appear and dissapear when thalamus is present

I want clear
"thalamus is present" group
"thalamus is not present" group


Like I want this corpse to be hidden when thalamus is present

**Comment by PanmanS** [comment]
![image-6](https://github.com/Regalis11/Barotrauma/assets/42521948/185ab83a-8b96-4a37-b283-5bd5b1fc82e9)

**Comment by PanmanS** [comment]
So what do you think?
